pyndl/preprocess.py

```diff
@@ -211,7 +222,19 @@ def create_event_file(corpus_file,
         be (three) consecutive words, a sentence, or a line in the corpus file.
 
     """
-    if '_' in symbols or '#' in symbols or '\t' in symbols:
+    try:  # assume symbols is a string
+        not_in_symbols = re.compile(f"[^{symbols:s}]")
+        def filter_symbols(line, replace): 
+            return not_in_symbols.sub(replace, line)
+    except TypeError:  # assume symbols is a filter function
+        def filter_symbols(line, replace):
+            line_copy = list(line)
+            for i in range(len(line)):
+                if not symbols(line[i]):
+                    line_copy[i] = replace
+            return ''.join(line_copy)
+
+    if filter_symbols('_#\t', replace='') != '':
```

Comment:
Add comment:
```# all special symbols has to be filtered out```
As I was thinking about this line for 3 minutes and only realized in the end, why it does the intended thing.

Reply:
After our style guide the variable for the for loop most likely should be `ii` instead if `i`.